Source/src/importers/ModelImporter.cpp

```diff
@@ -20,8 +20,11 @@ void Hachiko::ModelImporter::Import(const char* path)
     const aiScene* scene = nullptr;
     const std::filesystem::path model_path(path);
     const std::string model_output_path = StringUtils::Concat(model_path.parent_path().string(), "\\", model_path.filename().replace_extension(MODEL_EXTENSION).string());
-    //scene = import.ReadFile(path, aiProcess_Triangulate | aiProcess_FlipUVs | aiProcess_CalcTangentSpace);
-    scene = import.ReadFile(path, aiProcess_Triangulate | aiProcess_GlobalScale | aiProcess_CalcTangentSpace);
+
+    unsigned flags = aiProcess_CalcTangentSpace | aiProcess_GenSmoothNormals | aiProcess_JoinIdenticalVertices | aiProcess_ImproveCacheLocality | aiProcess_LimitBoneWeights
+                     | aiProcess_SplitLargeMeshes | aiProcess_Triangulate | aiProcess_GenUVCoords | aiProcess_SortByPType | aiProcess_FindDegenerates | aiProcess_FindInvalidData | 0;
```

Comment:
These GenSmoothNormals and GenUVCoords are not interfering with the texture's normals right?

Reply:
I don't think so.

Reply:
Okay 👌